cmd/admin-router.go

```diff
@@ -39,6 +39,9 @@ func registerAdminRouter(mux *router.Router) {
 	// Service update credentials
 	adminRouter.Methods("POST").Queries("service", "").Headers(minioAdminOpHeader, "set-credentials").HandlerFunc(adminAPI.ServiceCredentialsHandler)
 
+	// Info operations
+	adminRouter.Methods("GET").Queries("server-info", "").HandlerFunc(adminAPI.ServerInfoHandler)
```

Comment:
May be just 'info' it can have other params in future i.e if at all `info` gets an argument for example 

```
info=disk
```

Just returns only disk, currently since it is without any argument returns everything.